docs/articles_en/learn-openvino/large-language-models/ov-api-vs-optimum.rst

```diff
@@ -0,0 +1,132 @@
+.. {#native_vs_hugging_face_api}
+
+Hugging Face vs Native OpenVINO API
```

Comment:
The title is not aligned with what is shown on the left-hand side panel.

Reply:
thanks, aligned